src/react-components/src/components/03-organisms/Hero/Hero.tsx

```diff
@@ -0,0 +1,121 @@
+// OH-30 Header with Image Right
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "./HeroTypes";
+
+export interface HeroProps {
+  /** Can be Primary, secondary, tertiary, or 50/50. */
+  heroType?: HeroTypes;
+
+  /** Required heading element. */
+  heading: JSX.Element;
+
+   /** Optional subheader that displays underneath the
+    * required heading element.
+    */
+  subHeaderText?: JSX.Element;
+
+  /** Optional details area that contains location data. */
+  locationDetails?: JSX.Element;
+
+  /** Content creators can modify the foreground color
+    * when this component is used on Exhibition pages.
+    */
+  foregroundColor?: string;
+  /** Content creators can modify the background color
+    * when this component is used on Exhibition pages.
+    */
+  backgroundColor?: string;
+
+  /** Image used for primary Hero types. Note, cannot
+    * be used in conjunction with image.
+    */
+  backgroundImageSrc?: string;
+  /** Image used for secondary Hero types. Note, cannot
+    * be used in conjunction with backgroundImageSrc.
+    */
+  image?: JSX.Element;
+}
+
+export default function Hero(props: React.PropsWithChildren<HeroProps>) {
+
+  const heroBaseClass = "hero";
+
+  let {
+    heroType,
+
+    heading,
+
+    subHeaderText,
+
+    locationDetails,
+
+    foregroundColor,
+    backgroundColor,
+
+    backgroundImageSrc,
+    image,
+  } = props;
+
+  let heroModifiers;
+
+  if (heroType === HeroTypes.Primary) {
+    heroModifiers = ["primary"];
+  } else if (heroType === HeroTypes.Secondary) {
+    heroModifiers = ["secondary"];
+  }
```

Comment:
Should there be a fallback in case PRIMARY or SECONDARY aren't passed?

Reply:
I've instead made `heroType` required, so it'll now fail if a value from that enum isn't passed (it being optional was a holdover from when I was reworking this code).